source/2020-01-10-the-ember-times-issue-130.md

```diff
@@ -95,13 +95,28 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Where Can I Learn More? 🤓]((https://guides.emberjs.com/release/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+If you want to try out Ember Octane, you might want to learn more about how to use the new APIs and about best practices for
+building modern Ember apps. In this regard we highly recommend the following learning resources to you.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+### Getting Started
+
+- [The official documentation - Ember.js Guides](https://guides.emberjs.com/release/)
+- [The official beginner's tutorial - Build the Super Rentals app](https://guides.emberjs.com/release/tutorial/part-1/)
+- [The official API reference - Ember.js API Docs](https://api.emberjs.com/ember/release)
+
+### Octane Core Concepts
+
+- Octane Core Concepts Blog Series by [Chris Garrett (@pzuraq)](https://github.com/pzuraq): [Part 1](https://blog.emberjs.com/2019/02/11/coming-soon-in-ember-octane-part-1.html), [Part 2](https://blog.emberjs.com/2019/02/19/coming-soon-in-ember-octane-part-2.html), [Part 3](https://blog.emberjs.com/2019/02/26/coming-soon-in-ember-octane-part-3.html), [Part 4](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html) and [Part 5](https://blog.emberjs.com/2019/03/14/coming-soon-in-ember-octane-part-5.html)
```

Comment:
To help the reader know which link they might like to click, maybe:
```suggestion
- Octane Core Concepts Blog Series by [Chris Garrett (@pzuraq)](https://github.com/pzuraq)
  - [Part 1: Native Classes](https://blog.emberjs.com/2019/02/11/coming-soon-in-ember-octane-part-1.html)
  - [Part 2: Angle Brackets & Named Arguments](https://blog.emberjs.com/2019/02/19/coming-soon-in-ember-octane-part-2.html)
  - [Part 3: Tracked Properties](https://blog.emberjs.com/2019/02/26/coming-soon-in-ember-octane-part-3.html)
  - [Part 4: Modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html)
  - [Part 5: Glimmer Components](https://blog.emberjs.com/2019/03/14/coming-soon-in-ember-octane-part-5.html)
```
I'm not sure how the nested list will show up on the blog.

Reply:
Sounds great! I can check on the markup for the nesting locally as well